src/Formatter/AggregateMoneyFormatter.php

```diff
@@ -23,6 +23,10 @@
      */
     public function __construct(array $formatters)
     {
+        if (empty($formatters)) {
+            throw new \InvalidArgumentException(sprintf('Initialize an empty %s is not possible', self::class));
```

Comment:
Can you change `self::class` in `get_class($this)`? We are supporting PHP 5.5. Which I do not like, but until we are dropping it, we have to take care of it.

Reply:
I am not sure I understand.

http://php.net/manual/en/language.oop5.constants.php

> The special ::class constant are available as of PHP 5.5.0, and allows for fully qualified class name resolution at compile, this is useful for namespaced classes: 

Also the tests are passing for PHP 5.5 also

Reply:
Oh right. Damn. My bad.